### ISSUE TITLE
Dragging line do not work on touch device

### ISSUE THREAD

**Issue by moimael** [perliedman/leaflet-routing-machine#71]
Hi,

I tried to drag and drop lines to modify waypoints but it doesnt seems to work. Leaflet is the one catching the touch event. However if I double tap, it add a waypoint on the line, then i can drag it.

**Comment by perliedman** [comment]
Yes, you're right. If it's at all possible to hit the line, it's at least way too hard.

Non-desktop support is generally a bit lacking at the moment. For example, the itinerary would also need to work differently on small screens.
